### PR TITLE
Add fairseq to supported libraries for audio-to-audio

### PR DESCRIPTION
**Pull request by osanseviero**

### CODE REVIEW

**Review by Wauplin — APPROVED** [review]
Out of curiosity, what does it change? I expect https://huggingface.co/tasks/audio-to-audio to be updated. Does it impact inference tags as well?

File: packages/tasks/src/tasks/index.ts [hunk 1]
```diff
@@ -112,7 +112,7 @@ import type { ModelLibraryKey } from "../model-libraries";
  */
 export const TASKS_MODEL_LIBRARIES: Record<PipelineType, ModelLibraryKey[]> = {
 	"audio-classification": ["speechbrain", "transformers", "transformers.js"],
-	"audio-to-audio": ["asteroid", "speechbrain"],
+	"audio-to-audio": ["asteroid", "speechbrain", "fairseq"],
```
Comment (Wauplin) [inline packages/tasks/src/tasks/index.ts:115]:
```suggestion
	"audio-to-audio": ["asteroid", "fairseq", "speechbrain"],
```
(I like alphabetical order :smile:)

**Comment by osanseviero** [comment]
No, this only impacts the "compatible libraries" at top right https://huggingface.co/tasks/audio-to-audio

API is managed by https://github.com/huggingface/huggingface.js/blob/main/packages/tasks/src/library-to-tasks.ts#L22

**Review by SBrandeis — APPROVED** [review]

**Review by coyotte508 — APPROVED** [review]

**Review by julien-c — APPROVED** [review]

**Comment by osanseviero** [comment]
Merging as the error is unrelated

**Pull request merged**